@here/harp-mapview/lib/MapView.ts

```diff
@@ -535,7 +544,8 @@ export class MapView extends THREE.EventDispatcher {
 
     private readonly m_screenCamera = new THREE.OrthographicCamera(-1, 1, 1, -1);
 
-    private m_camera: THREE.PerspectiveCamera;
+    private readonly m_camera: THREE.PerspectiveCamera;
+    private readonly m_rtcCamera: THREE.PerspectiveCamera;
```

Comment:
I think this needs a comment about why we have two perspective camera's now.

Reply:
does rtc mean "relative to camera" or "relative to center"?

http://help.agi.com/AGIComponents/html/BlogPrecisionsPrecisions.htm at least it is referred to rte (i wonder if it makes sense to use this terminology instead?)